packages/api/src/external/commonwell/patient-updater-commonwell.ts

```diff
@@ -7,7 +7,7 @@ import { errorToString } from "../../shared/log";
 import { capture } from "../../shared/notifications";
 import { getPatients } from "../../command/medical/patient/get-patient";
 
-const maxNumberOfParallelRequestsToCW = 20;
+const maxNumberOfParallelRequestsToCW = 10;
```

Comment:
Discussed this with @leite08 on a huddle. 
Decided to lower this to avoid putting too much load on CW during EC.